src/pyscaffold/cli.py

```diff
@@ -169,7 +185,11 @@ def parse_args(args: List[str]) -> ScaffoldOpts:
         extension.augment_cli(parser)
 
     # Parse options and transform argparse Namespace object into common dict
-    return _process_opts(vars(parser.parse_args(args)))
+    opts = vars(parser.parse_args(args))
+    if opts["interactive"]:
```

Comment:
I suppose we can avoid branching here by relying on the dynamic behaviour of `dest="command"` for the`-i` flag...

Reply:
Smart, really smart!